shared/dataflow/codeql/dataflow/internal/FlowSummaryImpl.qll

```diff
@@ -54,6 +54,20 @@ signature module InputSig<LocationSig Location, DF::InputSig<Location> Lang> {
   /** Gets the return kind corresponding to specification `"ReturnValue"`. */
   Lang::ReturnKind getStandardReturnValueKind();
 
+  /**
+   * Gets the return kind corresponding to specification `"ReturnValue"` when
+   * the supplied argument `arg`.
```

Comment:
This doesn't make a lot of grammatical sense to me. Is there a verb missing somewhere?

Reply:
Oops. You're right. I've fixed it in 92e0b6430741a0f9b217e711d8fdab553494468b (hopefully 😅)